lib/ansible.sh

```diff
@@ -278,4 +278,60 @@ EOF
   )
 }
 
-set +a
\ No newline at end of file
+strap::ansible::playbook::run() {
+
+  local playbook_dir="$(pwd)/.strap/ansible/playbooks/default"
```

Comment:
You don't need to execute `pwd`.  Strap has `STRAP_WORKING_DIR` that you should reference directly (faster, lower overhead).

Reply:
Good to know! Changed